cmd/soci/commands/create.go

```diff
@@ -79,6 +81,14 @@ var CreateCommand = cli.Command{
 		}
 		spanSize := cliContext.Int64(spanSizeFlag)
 		minLayerSize := cliContext.Int64(minLayerSizeFlag)
+		// Creating the snapshotter's root path first if it does not exist, since this ensures, that
+		// it has the limited permission set as drwx--x--x.
+		// The subsequent oci.New creates a root path dir with too broad permission set.
+		if _, err := os.Stat(config.SociSnapshotterRootPath); os.IsNotExist(err) {
```

Comment:
Do we need to check for when `err != nil && !os.IsNotExist(err)`?

Reply:
I don't think so. It examines the error for type `os.ErrNotExist`. The null value, will force `os.IsNotExist` to return false, which is what we need.

Reply:
I meant, do we have to check for any errors from `os.Stat` besides `os.ErrNotExist`?

Reply:
Yeah, it does make sense. Updated.